### PR TITLE
YEN Stories – part 2 ‘The Welcome’

### PR DESCRIPTION
**Pull request by gabriel-Yen**
**YEN Stories** – part 2 ‘The Welcome’
dedicated to A^^

> **_One picture is worth a thousand words_** - a Chinese proverb

In this topic, I would like to share an idea of necessity to have some sort of ‘The Welcome’ for new YENizens. 

My value proposition or rather request is to team really consider the creation of a welcoming video and through that, what is the most important - show our great community. Our Yenizens, our Yenfamily – Us…

I mean… how great that could be if, in the welcoming post, we would post the link to a video, where our new friends would see:

- warm welcome from the CEO – Yes, John :) a few words
- Peter shouting Yen.ioooo :) 
- a short introduction from each team member, ‘what you do – responsibilities, why you are here but in the team way of sorts, professional but also fun, like for example James – I’m here for the burgers :)
- a few seconds clips – literally, like shots, from our Yenizens
- short invite to transact, showing 3 clicks transaction.

Everything combines with great music, a few minutes video – can’t be a long but a solid and really dynamic introduction to our world.  

Why We need that? 

So this is very simple for us and early members, those being around the Pub and DCTV, they know of everything, they know possibilities of the platform but how about those masses who will come to YEN when the platform will be open for everyone? That’s kind of rhetorical question, isn't?   

Let’s show them the way, let’s show them our great community, sharing why we are doing this, Why we are here and what they can expect by being around us.

That could be a very first step for ‘Yen Stories’ and ‘People of Yen’…and the best way to do it is put together worm welcome from the team and many short clips from our members saying simple “hello, Yo, what’s up e.t.c. with a short description to their profile.  

For alpha and beta cohorts that would be a big promotion itself but also to encourage those members who don't post anything on their YouTube, we could persuade them to take part in ‘The Welcome’ by participation in the giveaway for Yenizens only. 

I hope that the sentence above doesn't sound harsh.

-----
In conclusion, ‘The welcome’ could be a great tool for introduction as well as follow up tool in the platform, especially now when Paul joined the team and the community manager has coming...
I believe that would be a good area to create this valuable project and also experiment…

because it is very important to keep experimenting, right? Do it faster, execute to learn and win...

### CODE REVIEW

**Comment by saddington** [comment]
Closing this as we've moved to Trello! Great stuff @gabriel-Yen !
